pkg/providers/tinkerbell/stack/stack.go

```diff
@@ -140,6 +127,10 @@ func (s *Installer) Install(ctx context.Context, bundle releasev1alpha1.Tinkerbe
 			deploy: !s.bootsOnDocker,
 			image:  bundle.Boots.Image.URI,
 			env:    bootEnv,
+			args: []string{
+				"-dhcp-addr=0.0.0.0:67",
```

Comment:
it doesn't default to that. The reason why we didn't have it previously was because we weren't providing custom args for boots. If you look in the helm values for boots, the args defaults to `dhcp-addr=0.0.0.0:67` if not provided https://github.com/aws/eks-anywhere-build-tooling/blob/main/projects/tinkerbell/tinkerbell-chart/chart/values.yaml#L14